engine-comparison/runner.sh

```diff
@@ -30,6 +30,9 @@ if [[ $FUZZING_ENGINE == "afl" ]]; then
   if [[ !$(find seeds -type f) ]]; then
     echo "Input" > ./seeds/nil_seed
   fi
+  # TODO: edit core_pattern in Docker VM
+  # https://groups.google.com/forum/m/#!msg/afl-users/7arn66RyNfg/BsnOPViuCAAJ
+  export AFL_I_DONT_CARE_ABOUT_MISSING_CRASHES=1
```

Comment:
Does this suppress crashes that we want to detect?

Reply:
I also wasn't sure, I discussed this with @kcc and it sounds fine for now because it works

Reply:
So it can still detect all current benchmarks?

Reply:
I'm not sure, I found that it was necessary to begin fuzzing, but I haven't yet tested it to completion on all or even most benchmarks